pkg/providers/nutanix/config/cp-template.yaml

```diff
@@ -0,0 +1,196 @@
+piVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: NutanixCluster
+metadata:
+  name: "{{.clusterName}}"
+  namespace: "{{.eksaSystemNamespace}}"
+spec:
+  prismCentral:
+    address: "{{.nutanixEndpoint}}"
+    port: {{.nutanixPort}}
+    insecure: true
+    credentialRef:
+      name: "{{.clusterName}}"
+      kind: Secret
+  controlPlaneEndpoint:
+    host: "{{.controlPlaneEndpointIp}}"
+    port: 6443
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: "{{.clusterName}}"
+  namespace: "{{.eksaSystemNamespace}}"
+stringData:
+  NUTANIX_PASSWORD: "{{.nutanixPassword}}"
+  NUTANIX_USER: "{{.nutanixUser}}"
```

Comment:
Instead of using `stringData`, can we just use `data` and base64 encode the creds? This ensures the creds are not displayed in plain text and also handles any special characters that the creds might have.
Here's an example of how we do it for other creds https://github.com/aws/eks-anywhere/blob/main/pkg/providers/vsphere/config/secret.yaml#L8-L9

Reply:
We will be redoing the creds in another PR (including the format) in a week or so hopefully based on the standardized creds format we use in other k8s distros. Can we punt on this for now?

Reply:
Could you raise an issue to address explicitly so we don't miss. Thanks.